docs/design-ux/editor.md

```diff
@@ -0,0 +1,37 @@
+# Editor
+When a user has a document selected, that file is open for editing in an editor. The name of the document is displayed above the document and is editable (editing renames the document).
```

Comment:
Depending on the UI platform, this could be challenging, I agree it's worth doing.